tests/unit/cloud/clouds/test_hetzner.py

```diff
@@ -0,0 +1,448 @@
+"""
+    :codeauthor: Florian Kantelberg <florian.kantelberg@initos.com>
+"""
+
+from salt.cloud.clouds import hetzner
+from salt.exceptions import SaltCloudException, SaltCloudSystemExit
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.mock import MagicMock, patch
+from tests.support.unit import TestCase, skipIf
+
+KEY = "abcdefgh"
+
+VM_NAME = "myserver"
```

Comment:
We try to stray away from these module level variables as they can cause headache in the future and hard to find if anyone ever ends up importing this test module. 

Also another ask, we have recently migrated to pytest. Would you mind migrating this test to `tests/pytests/unit/cloud/`. You will need to create the cloud directory since there isn't currently one there. thanks :)